packages/app/src/app/pages/Dashboard/Components/TemplatesRow/TemplatesRow.tsx

```diff
@@ -43,110 +77,61 @@ export const TemplatesRow: React.FC = () => {
         collectionId: collection?.id,
       },
     });
-
-    actions.modals.newSandboxModal.close();
   };
 
   return (
-    <Stack
-      as="section"
-      css={css({
-        width: `calc(100% - ${2 * GUTTER}px)`,
-        marginX: 'auto',
-      })}
-      direction="vertical"
-      gap={4}
-    >
-      <Text
-        as="h2"
-        css={css({
-          margin: 0,
-        })}
-        weight="400"
-        size={4}
-      >
-        Start from a template or import from GitHub
-      </Text>
-      <TemplatesGrid>
-        {officialTemplates.state === 'loading' &&
-          new Array(5).fill(undefined).map((_, i) => (
-            <SkeletonText
-              // eslint-disable-next-line react/no-array-index-key
-              key={`templates-skeleton-${i}`}
-              css={css({
-                width: '100%',
-                height: '116px',
-              })}
-            />
-          ))}
-
-        {officialTemplates.state === 'ready' && (
-          <>
-            {officialTemplates.templates
-              .slice(0, 3) // TODO: define which templates to show
-              .map(template => (
+    <Stack direction="vertical" gap={6}>
+      {title !== undefined && title !== '' ? (
+        <EmptyPage.StyledGridTitle as="h2">
+          Start from a template
+        </EmptyPage.StyledGridTitle>
+      ) : null}
+      <EmptyPage.StyledGrid as="ul">
+        {officialTemplates.state === 'loading'
+          ? new Array(templateIds.length).fill(undefined).map((_, i) => (
```

Comment:
Now that we're using `templateIds.length` we can just map over `templateIds`, right? Do we still need this loading state and will we still need to fetch `officialTemplates` or are we just using hardcoded templates?

Reply:
@tristandubbeld are you suggesting we hardcode everything related to the templates rather than using the selected ids to find the full info after fetching the official templates?

Reply:
Hmm yeah didn't think about it that way, wasn't aware most information was fetched - doing it like this makes most sense! We do still need the loading state then, but we can map over the id's instead of filling an array with `undefined`.

Reply:
@tristandubbeld makes sense! I'll update it :)

Reply:
[2b48d29](https://github.com/codesandbox/codesandbox-client/pull/7234/commits/2b48d29b09b7ab80b6b35436002d3027e21bfb55)